server/rpm/pbench-server.spec

```diff
@@ -110,31 +108,33 @@ rm -rf Makefile %{static}/Makefile \
    lib/%{pyversion}/site-packages/* \
    lib/pbench/common/__pycache__ \
    lib/pbench/cli/__pycache__ \
-   lib/pbench/cli/getconf.py \
    lib/pbench/cli/agent \
    lib/pbench/server/__pycache__
 
 %post
 
 chown -R pbench.pbench /%{installdir}
 
+cd /%{installdir}
+
 # on RHEL7 we use scl to get python3
 %if 0%{?rhel} == 7
 # install python3-elasticsearch
-echo "scl enable rh-python36 'pip install elasticsearch1 boto3 ujson'"
-scl enable rh-python36 'pip install elasticsearch1 boto3 ujson'
+echo "scl enable rh-python36 -- python3 -m pip install -r requirements.txt"
+scl enable rh-python36 -- python3 -m pip install -r requirements.txt
 %endif
 
 %if 0%{?rhel} == 8
 # install python3-elasticsearch
-echo "pip3 install elasticsearch1 urllib3 boto3 ujson"
-pip3 install elasticsearch1 urllib3 boto3 ujson
+echo "python3 -m pip install -r requirements.txt"
+python3 -m pip install -r requirements.txt
 %endif
 
 # on Fedora, we need to pip-install elasticsearch1: only the latest
 # version is packaged as an RPM in the Fedora repos.
 %if 0%{?fedora}
-pip3 install elasticsearch1 urllib3 boto3 ujson
+echo "python3 -m pip install -r requirements.txt"
```

Comment:
Any way to make this be `%if 0%{?rhel} == 8 or 0%{?fedora}`?

Reply:
I've left them separate so far because in some case in the past, rhel8 and fedora needed different treatment. I'll squash them at the very end, if they remain the same until that point.